fabfile.py

```diff
@@ -391,12 +389,15 @@ def validate(c):
     user = get_value(c, "application", "user")
     wif = get_value(c, "application", "workload_dir") + f"/{workload}/{workload}.sh"
     command = get_env(c, "environment") + " bash " + wif + " validate"
+    failure_file = get_value(c, "application", "failure_file") + "/output.txt"
 
     result = c.sudo(command, user=user, warn=True)
-    failure_file = get_value(c, "application", "failure_file") + "/output.txt"
     if not result:
         raise Exit(f"Validate failed for '{workload}' workload.")
-
+    
+    # Remove the validate output file on success. 
+    c.sudo(f"rm {failure_file}", user=user, warn=True)
```

Comment:
I don't think we need to do that here. We can do it in the bash script directly, I will add a comment there.